django_x509/base/admin.py

```diff
@@ -119,6 +120,16 @@ def crl_view(self, request, pk):
                             status=200,
                             content_type='application/x-pem-file')
 
+    def renew_ca(self, request, queryset):
+        renewed_rows = 0
+        for ca in queryset:
+            ca.renew_ca()
+            renewed_rows += 1
+        message = _('{0} CAs have been successfully renewed'.format(renewed_rows))
```

Comment:
please use pluralization to display the correct message depending on the amount of objects on which the operation was performed, see here: https://docs.djangoproject.com/en/3.0/topics/i18n/translation/#pluralization

Follow the exact style used in that example.